frontend/src/pages/events/index.js

```diff
@@ -7,29 +7,42 @@ import SundayCelebBarEvents from "../../components/page-events/sundayCelebBarEve
 import { StaticImage } from "gatsby-plugin-image";
 import EventCard from "../../components/page-events/eventCard";
 
-const EventsPage = () => (
-  <Layout>
-    <SundayCelebBarEvents />
-    <div>Event list section</div>
-    <EventCard
-      title="Prayer Gathering"
-      date="Th, May 25, 2023"
-      img={
-        <StaticImage
-          className="h-[6.15788rem] w-[10.94738rem] flex-shrink-0 mb-0 md:h-60 md:w-[26.66669rem]"
-          src="../../images/prayer-gathering.png"
-          alt="Background"
-        />
-      }
-      location="T Center"
-      description="I'm baby migas fam yuccie, big mood freegan affogato everyday
-          carry hashtag four dollar toast truffaut 3 wolf moon beard. Hella
-          live-edge brunch neutral milk hotel pabst."
-    />
-    <div>Prayer Gatherings bar</div>
-    <Instagram />
-  </Layout>
-);
+const EventsPage = () => {
+  const events = [{
+    id: 1,
+    title: "HMCC Event",
+    date: "Th, May 25, 2023",
+    imgUrl: "../../images/prayer-gathering.png",
+    imgAlt: "example",
+    location: "TCenter",
+    description: "event at Hmcc"
+  }] // Todo: fetch actual events from backend
+
+  return (
+    <Layout>
+      <SundayCelebBarEvents />
+      <div>Event list section</div>
```

Comment:
Can you make this into a different component (like `SundayCelebBarEvents`, `Instagram`) and put it into the components page?

Reply:
and can put the `const events` list into the new component as well; we'll work on connecting to the backend soon.